### ISSUE TITLE
[mac] Metro Bundler error in launchPackager.command

### ISSUE THREAD

**Issue by RandyHBH** [electrode-io/electrode-native#705]
Hi, guys, I update my electrode-native from v0.14.3 to v0.16.0, and I am trying to test my mini app on my iPad and when I run `ern run-ios --host 192.168.1.240 ` the Metro Bundler is raising this error

```
Last login: Thu Mar 29 13:21:20 on ttys001
mac-mini:~ aleph$ /Users/aleph/.ern/containergen/out/ios/ElectrodeContainer/Libraries/ReactNative/scripts/launchPackager.command ; exit;

module.js:540
    throw err;
    ^
Error: Cannot find module '/Users/aleph/.ern/containergen/out/ios/ElectrodeContainer/Libraries/ReactNative/local-cli/cli.js'
    at Function.Module._resolveFilename (module.js:538:15)
    at Function.Module._load (module.js:468:25)
    at Function.Module.runMain (module.js:684:10)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3
Process terminated. Press <enter> to close the window
```
and the app is still searching for the server in the wrong address ( localhost:8081 ) even when I specify another IP.

What is happening? Any Help?

**Comment by clairewli** [comment]
Hey @RandyHBH , thanks for opening the issue. From the error you see, it's some noise from react-native. We see it even when we are using localhost:8081 and it doesn't prevent us from running the miniapp.

**Comment by clairewli** [comment]
@RandyHBH  you could manually set your local host here for now https://github.com/electrode-io/electrode-native/blob/master/ern-container-gen/src/generators/ios/hull/ElectrodeContainer/ElectrodeReactNative.m#L54.

**Comment by clairewli** [comment]
I tried the command with earlier version, I think it's introduced in 0.16.0. so it may take some time for us to get back to it and fix it.

**Comment by RandyHBH** [comment]
@clairewli Thanks, i already change it manually :), and it works.

**Comment by belemaire** [comment]
@RandyHBH 
As stated in [v0.16.0 release notes](https://github.com/electrode-io/electrode-native/releases/tag/v0.16.0), the `--host` option is not functional yet.
We have one open issue https://github.com/electrode-io/electrode-native/issues/688 to finalize implementation, unfortunately we were not able to ship it in 0.16.0, it will be released in 0.17.0. Should have rather not ship the option as it was only partially developed and not final. Our bad. Sorry for the confusion it created :(

**Comment by RandyHBH** [comment]
@belemaire do not worry about it, just try to make it more visible to distracted people like me :)
